modelforge/curation/spice_curation.py

```diff
@@ -0,0 +1,256 @@
+from modelforge.utils.units import *
+
+from modelforge.curation.curation_baseclass import *
+
+
+class SPICE_1_1_4_curation(dataset_curation):
```

Comment:
Is it possible to define a SpiceCuration(DatasetCuration) that takes as keyword argument the specific version we want to pull down?

Reply:
I think we can.  I'll have to check with Peter Eastman on what the specific differences are and whether that is actually something useful (I think e.g., 1.1.2 is just a subset of the 1.1.4 dataset).  It would be an easy chance to make in the future if that is ultimately relevant.  Our interest is mostly on the openff level theory for spice, that is only available from qcarchive at the moment.